File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/infrastruktur/integrasjoner/enhetsregisteret/EnhetsregisteretMappingException.java
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.infrastruktur.integrasjoner.enhetsregisteret;

public class EnhetsregisteretMappingException extends RuntimeException {
    EnhetsregisteretMappingException(String msg, Exception e) {
        super(msg, e);
    }

    EnhetsregisteretMappingException(String msg) {
        super(msg);
    }
}

File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/infrastruktur/enhetsregisteret/EnhetsregisteretMappingException.kt
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.infrastruktur.enhetsregisteret;

public class EnhetsregisteretMappingException extends RuntimeException {
  EnhetsregisteretMappingException(String msg, Exception e) {
    super(msg, e);
  }

  EnhetsregisteretMappingException(String msg) {
    super(msg);
  }
}
